modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -2404,8 +2582,10 @@ Besides the default `--help` option, this command also takes the option `--defau
 ----
 > cloudhub vpc business-groups add [options] <vpc> <businessGroups...>
 ----
-This command assigns the Anypoint VPC defined in `<vpc>` to the business group(s) passed as argument(s) thereafter. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command assigns the Anypoint VPC defined in `<vpc>` to the business group(s) passed as argument(s) thereafter.
```

Comment:
Do we allow constructions like "group(s)" and "argument(s)" ?

Reply:
"passed thereafter" works for me. Not sure if we normally use it.

Reply:
Nope to the first.